website/content/docs/job-specification/service.mdx

```diff
@@ -93,6 +98,10 @@ Service Mesh][connect] integration.
   provider supports `tcp` and `http` checks. The Consul integration supports
   the `grpc`, `http`, `script`<sup><small>1</small></sup>, and `tcp` checks.
 
+- `weights` <code>([Weights][weights]: nil)</code> - Specifies how a service
+  instance is weighted in a DNS SRV request based on the service's health status.
+  Only available where `provider = "consul"`.
```

Comment:
It's good to have a link to the Consul docs but to allow for drift (and to account for the subtle difference between block and map syntax) let's inline the field definitions:

```suggestion
- `weights` <code>(Weights: nil)</code> - Specifies how a service instance is
  weighted in a DNS SRV request based on the service's health status, as
  described in the Consul [weights][] documentation. Only available where
  `provider = "consul"` The `weight` block supports the following fields:
  - `passing` <code>int: 1</code> - The weight of services in passing state.
  - `warning` <code>int: 1</code> - The weight of services in warning state.

```